servers.json

```diff
@@ -35,6 +35,12 @@
       "max_restarts": "1"      
     },
     {
+      "name": "content-server basket proxy PORT 3031",
```

Comment:
when i run this manually from one of the PRs, I get ` FxA Basket Proxy listening on port 1114`, not port 3031